Sprint-1/exercises/random.js

```diff
@@ -2,8 +2,12 @@ const minimum = 1;
 const maximum = 100;
 
 const num = Math.floor(Math.random() * (maximum - minimum + 1)) + minimum;
-
 // In this exercise, you will need to work out what num represents?
 // Try breaking down the expression and using documentation to explain what it means
 // It will help to think about the order in which expressions are evaluated
 // Try logging the value of num and running the program several times to build an idea of what the program is doing
+  
+     // The variable num represents a random integer between 1 and 100.
+    //  so each time you run the program, a different integer within the range [1, 100] will be generate.
```

Comment:
To test your understanding, how would you write an expression 
(without using any variable in the expression) that yields a random 
integer between -5 and 3 (including -5 and 3)?

Reply:
Math.floor(Math.random() * (3 + 5 + 1)) - 5;